test/swap_test.ts

```diff
@@ -640,11 +641,18 @@ function expectCorrectQuoteResponse(
         }
 
         if (prop === 'debugData') {
-            const { samplerGasUsage, ...rest } = quoteResponse[property];
-            const { samplerGasUsage: expectedSamplerGasUsage, ...expectedRest } = expectedResponse[property];
+            const { samplerGasUsage, blockNumber, ...rest } = quoteResponse[property];
+            const {
+                samplerGasUsage: expectedSamplerGasUsage,
+                blockNumber: expectedBlockNumber,
+                ...expectedRest
+            } = expectedResponse[property];
             console.log(samplerGasUsage, expectedSamplerGasUsage);
+            console.log(blockNumber, expectedBlockNumber);
```

Comment:
nit: delete?

(I forgot to delete the one for `samplerGasUsge`)

Reply:
ack